certification-content-bundle-spec.md

```diff
@@ -0,0 +1,83 @@
+# Qwiklabs Certification Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: Certification
+schema_version: 1
+
+default_locale: en
+
+title: 
+  locales:
+    en: Virtual Machines in GCP
+
+description:
+  locales:
+    en: Prove what you know about VMs in GCP by doing these courses and exams!
+
```

Comment:
As part of the basic attributes we should discuss which fields from the sample we want to carry forwards: https://github.com/CloudVLab/qwiklabs-test-content/blob/master/certifications/bec-certificate/qwiklabs.yaml

- objectives
- audience
- prerequisites
- tags
- product_tags
- role_tags
- domain_tags
- credits
- certificate_award